packages/react-router/docs/guides/migrating.md

```diff
@@ -0,0 +1,141 @@
+# Migrating from v3 to v4
```

Comment:
I would lump v2 into things too, since it has the same API with some deprecation warnings.